### DISCUSSION TITLE
Tracking registration date in FHIR Application

### DISCUSSION THREAD

**Discussion by rkodev** [opensrp/fhircore#396] [Ideas]
How to track registration dates in FHIR Application.

We currently need to track the registration date of a Patient. One of the suggested methods is to use [Encounter](https://www.hl7.org/fhir/encounter.html) for this purpose, i.e generate an Encounter immediately after registration.

@pld @ekigamba @ndegwamartin @Rkareko

**Comment by pld** [comment]
Yea that could make sense, what other alternative ways are their to track this date? (so we can compare)

And if we use Encounter, do you have a sense of how it'd fit into the resource graph? Kinda makes sense to rethink the interaction cycle around Encounter, since from the description they're a standard piece

**Comment by f-odhiambo** [comment]
Tracking of registration date can be retrieved from history and not an encounter  tracked here #474

**Reply by pld** [reply]
Is it normal to have a set of standard encounters that are pre-defined? Like the typical health use-case for health service of type A is the progression through encounter graph B
